src/synthesizer/golang/mod.rs

```diff
@@ -620,25 +620,37 @@ impl GolangEmitter for ResourceIr {
                 }
             }
             Self::Object(structure, properties) => {
+                let mut structure_is_simple_json = false;
                 let props = output.indent_with_options(IndentOptions {
                     indent: INDENT,
                     leading: Some(match structure {
                         Structure::Composite(name) => match *name {
                             "Tag" => "&cdk.CfnTag{".into(),
                             name => format!("&{name}/* FIXME */{{").into(),
                         },
-                        Structure::Simple(cfn) => {
-                            unreachable!("object with simple structure ({:?})", cfn)
+                        Structure::Simple(cfn) => match cfn {
+                            CfnType::Json => {
+                                structure_is_simple_json = true;
+                                "map[string]interface{} {".into()
+                            }
+                            _ => unreachable!("object with simple structure ({:?})", cfn)
                         }
                     }),
                     trailing: Some("}".into()),
                     trailing_newline: false,
                 });
                 for (name, val) in properties {
-                    props.text(format!(
-                        "{name}: ",
-                        name = golang_identifier(name, IdentifierKind::Exported)
-                    ));
+                    if structure_is_simple_json { 
+                        props.text(format!(
+                            "\"{name}\": ",
+                            name = golang_identifier(name, IdentifierKind::Exported)
+                        ));
+                    } else {
+                        props.text(format!(
+                            "{name}: ",
+                            name = golang_identifier(name, IdentifierKind::Exported)
+                        ));
+                    } 
```

Comment:
This is required because the format string for the keys needs to be different depending on if it's for a "real" type or a just a json blob. Json blobs are represented as `map[string]interface{}` in go, which means the keys needs to be wrapped in double quotes `"`.

The way it is written is pretty verbose, and I am new to rust so please let me know if there is a more concise way to write it! I was hoping to do something like the below, 
but this doesn't work because the format string in the `format!` macro MUST be a string literal. There is no way to define it dynamically that I have found. 
```
props.text(format!(
    if structure_is_simple_json { "\"{name}\": " } else { "{name}: " },
    name = golang_identifier(name, IdentifierKind::Exported)
));
```